docs/persistent-subscriptions/README.md

```diff
@@ -0,0 +1,93 @@
+# Persistent subscriptions
+
+A common operation is to subscribe to a stream and receive notifications for changes. As new events arrive, you continue following them. 
+
+You can only subscribe to one stream or the $all stream. You can use server-side projections for linking events to new aggregated streams. System projections create pre-defined streams that aggregate events by type or by category and are available out-of-the box. Check the server documentation to learn more about system and user-defined projections.
+
+Persistent subscriptions run on the Leader node and are not dropped when the connection is closed. Moreover, this subscription type supports the "[competing consumers](https://www.enterpriseintegrationpatterns.com/patterns/messaging/CompetingConsumers.html)" messaging pattern and are useful when you need to distribute messages to many workers. EventStoreDB saves the subscription state server-side and allows for at-least-once delivery guarantees across multiple consumers on the same stream. It is possible to have many groups of consumers compete on the same stream, with each group getting an at-least-once guarantee.
+
+::: tip
+The Administration UI includes a _Persistent Subscriptions_ section where a user can create, update, delete and view subscriptions and their statuses.
```

Comment:
**COMMENT:** Might be worth adding a note about not possible to add the persistent subscription to all from the UI.

Reply:
I agree, I've added a comment about it in the tip